docker-plugin/src/main/java/com/nirima/jenkins/plugins/docker/DockerTemplateBase.java

```diff
@@ -45,7 +46,7 @@
     public final String hostname;
 
     public final String[] dnsHosts;
-    public final String[] volumes;
+    public final String volumesString;
```

Comment:
Better store in real format and return String only in expected for UI getter. This must simplify many code.